lib/statsample-glm/glm/formula.rb

```diff
@@ -29,9 +29,11 @@ def parse_formula(form = :token)
 
       private
 
-      def add_contant_term_if_required
-        # TODO: Add support for constants
-        @tokens.unshift Token.new('1')
+      def manage_contant_term
+        @tokens.unshift Token.new('1') unless
+          @tokens.include?(Token.new('1')) ||
+          @tokens.include?(Token.new('0'))
+        @tokens.delete Token.new('0')
```

Comment:
As far as I can see, that's because `Token` class is defined in the same file where you are trying to define your constant, but it is defined **after** constant definition. So, when Ruby evaluates `TOKEN_1 = Token.new('1')`, it knows absolutely nothing about `Token` class existance. You can solve this by moving entire `Token` class above `FormulaWrapper`.